.docker/entry-point.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+set -e
+
+# Set the apache user and group to match the host user.
+OWNER=$(stat -c '%u' /data)
+GROUP=$(stat -c '%g' /data)
+if [ "$OWNER" != "0" ]; then
```

Comment:
Both are created during the build fase of the container. In all unix systems a group is removed when all members are removed.